application/src/main/java/org/opentripplanner/routing/api/request/preference/AccessEgressPreferences.java

```diff
@@ -171,7 +171,9 @@ private static TimeAndCostPenaltyForEnum<StreetMode> createDefaultCarPenalty() {
     // Add penalty to all car variants with access and/or egress.
     var carPenalty = TimeAndCostPenalty.of(TimePenalty.of(ofMinutes(20), 2f), 1.5);
     for (var it : StreetMode.values()) {
-      if (it.includesDriving() && (it.accessAllowed() || it.egressAllowed())) {
+      if (
+        it.includesDriving() && (it.accessAllowed() || it.egressAllowed()) && !it.transferAllowed()
+      ) {
```

Comment:
```suggestion
      // Apply car-penalty to all car modes allowed in access/egress only. Exclude car modes(CAR) used
      // in direct street routing and car modes used when you bring the car with you onto transit. This is 
      // a bit limited and will not work if we combine car access/egress modes like CAR_TO_PARK with CAR
      // in the same search. This currently not possible, but if we enable this in the future this logic must be 
      // looked at again.
      if (
        it.includesDriving() && (it.accessAllowed() || it.egressAllowed()) && it != CAR
      ) {
```

Reply:
I changed it to this and removed the existing comment